src/core/connection.c

```diff
@@ -6479,6 +6511,7 @@ QuicConnParamSet(
         //
         // Must be set before the client connection is started.
         //
+
```

Comment:
nit: remove this new line and add back the one you removed below.

Reply:
Done